CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+
+### Request a final review
+
+1. Before requesting a final review, [squash all commits on your feature branch into one](#squashing-all-commits-into-one). This will consolidate all your work and include a well-formed commit message to be used in the changelog.
+1. After pushing the squashed commits, mention the `@blackbaud/skyux-maintainers` to review the complete contribution. We review the implementation, testing, functionality, styling, localization, and accessibility.
+
+#### Addressing review feedback
+
+1. Make required changes to the source code.
+1. Rerun all test suites.
+1. Create a "fixup" commit:
+
+```
+git commit --all --fixup HEAD
+git push
+```
+
+#### Updating the commit message
+
+Sometimes, the reviewer will ask that you reword your commit message. You can do so using the `--amend` flag:
+
+```
+git commit --amend
+git push --force-with-lease
+```
+
+_Note: If you need to ammend an earlier commit, you'll need to [squash all commits into one](#squashing-all-commits-into-one)._
+
+#### Squashing all commits into one
+
+If your pull request branch has several non-fixup commits, you'll need to squash all of the commits into one commit with a message that meets the [`conventional-changelog` specification](https://github.com/conventional-changelog/conventional-changelog).
+
+Run the following command to squash all commits in your feature branch with a new commit message that you provide. While local changes will remain intact, it does rewrite your feature branch's commit history.
+
+```
+npm run dev:pristine
+```
+
+## <a name="coding-rules"></a> Coding Rules
+
+1. All features and bug fixes should include at least one test spec and should not lower the overall code coverage percentage.
+1. All public API component inputs and outputs and public service methods should be documented with JSDocs comments.
+1. Code should be styled using our Prettier rules.
+
+```
+npm run dev:format
+```
+
+## <a name="commit-message-guidelines"></a> Commit Message Guidelines
+
+- We strictly follow [Angular's commit message format](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format).
+- A full list of supported **scopes** can be found [here](https://github.com/blackbaud/skyux/blob/main/.github/workflows/validate-pr.yml).
+- Run the following command to create a commit message using the [`conventional-changelog` specification](https://github.com/conventional-changelog/conventional-changelog):
+
+```
+npm run dev:commit
+```
+
+## <a name="local-development"></a> Local Development Cheatsheet
```

Comment:
```suggestion
## <a name="local-development"></a> Local development cheat sheet
```
Do we need to add this to the docs on the website?

Reply:
No, this is specific to the monorepo.